examples/nodejs/token-vending-machine/lambda/config.ts

```diff
@@ -41,4 +45,16 @@ export const tokenPermissions: TokenScope = {
  * days, or using epoch format. You may also set tokens to never expire.
  * More information here: https://docs.momentohq.com/develop/api-reference/auth-tokens#generateauthtoken-api
  */
-export const tokenExpiresIn: ExpiresIn = ExpiresIn.hours(1);
\ No newline at end of file
+export const tokenExpiresIn: ExpiresIn = ExpiresIn.hours(1);
+
+/**
```

Comment:
let's create an enum called AuthenticationMethod just above your comments here, and provide the three legal values. That way it's not possible for a user to configure it to an invalid value.